UnsplashPhotoPicker/UnsplashPhotoPicker/Classes/Controllers/UnsplashPhotoPickerViewController.swift

```diff
@@ -199,10 +189,6 @@ class UnsplashPhotoPickerViewController: UIViewController {
         ])
     }
 
-    private func setupPeekAndPop() {
-        previewingContext = registerForPreviewing(with: self, sourceView: collectionView)
-    }
-
```

Comment:
Please let's try to reduce the number of changes... can we leave this method as is?